pkg/utils/landscaper/blueprint_test.go

```diff
@@ -0,0 +1,33 @@
+package landscaper
+
+import (
+	"testing"
+
+	"github.com/mandelsoft/vfs/pkg/osfs"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestConfig(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Landscaper utils Test Suite")
+}
+
+var _ = Describe("Landscaper", func() {
+
+	Context("Render blueprint", func() {
+
+		It("should render a blueprint with a target list import", func() {
```

Comment:
can you also add a test for a component descriptor import

Reply:
I have added a second test for imports of a component descriptor and a component descriptor list.